_posts/2020-10-09-torch-optim/torch_network_with_optim.Rmd

```diff
@@ -0,0 +1,269 @@
+---
+title: "Optimizers in torch"
+description: > 
+ Today, we wrap up our mini-series on torch basics, adding to our toolset two abstractions we hadn't made use of before: loss functions and optimizers.
+
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanatorchnetworkfromscratch
+date: 10-01-2020
```

Comment:
If I'm understanding things correctly, I think this is supposed to be the ninth of October.

Reply:
thanks for spotting this!